lib/Map/RegionProvider.js

```diff
@@ -200,39 +240,32 @@ RegionProvider.prototype.loadRegionIDs = function() {
         throw (new DeveloperError('No server for region mapping defined: ' + this.regionType));
     }
     var that = this;
-
-    var baseuri = URI(this.server).addQuery({
-        service: 'wfs',
-        version: '2.0',
-        request: 'getPropertyValue',
-        typenames: this.layerName});
-
-    // get the list of IDs that we will attempt to match against for this column
-    var promises = [];
-    var url = baseuri.setQuery('valueReference', this.regionProp).toString();
-
-    if (corsProxy.shouldUseProxy(url)) {
-        url = corsProxy.getURL(url);
+    var p;
+    if (defined(this.regionIdList)) {
+        p = loadJson(this.regionIdList);
+    } else {
+        p = this.loadRegionsFromWfs(this.regionProp);
     }
+    p.then(function(json) {
+        return that.processRegionIds(json.values, undefined, 'serverReplacements');
+    }).otherwise(function(err) {
+        console.log(err);
+        throw(err);
+    });
+    var promises = [p];
 
-    promises.push(loadText(url)
-        .then(function(xml) {
-            that.loadRegionsFromXML(xml, undefined, "serverReplacements");
-        }).otherwise(function(err) {
-            console.log(err);
-            throw(err);
-        }));
     // if this column might be ambiguous then fetch the disambiguating values for each column as well (usually State)
     if (this.disambigProp) {
-        url = baseuri.setQuery('valueReference', this.disambigProp).toString();
-        promises.push(loadText(url).then(function(xml) {
-            that.loadRegionsFromXML(xml, that.disambigProp, "disambigServerReplacements");
-        }));
+        var dp = this.loadRegionsFromWfs(this.disambigProp);
```

Comment:
Wondering about this line - as you say, loading from wfs is the slower option - is there a way to tell if the faster json option is available for the disambiguation variable? I imagine it's hard because this RegionProvider would need to know things about other regions.